recipes-tag/lightflow/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.10.2" %}
+
+package:
+    name: lightflow
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/australiansynchrotron/lightflow
+    git_rev: v{{ version }}
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy x.x
```

Comment:
Do we need `x.x` here at all? Same for line 23.

Reply:
done. thanks!